packages/terra-site/src/examples/form-input/controlled_examples/DefaultExample.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import FormInput from 'terra-form-input/lib/Input';
+
+class DefaultExample extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { input: 'Default Input' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ input: event.target.value });
+  }
```

Comment:
nit: Looks like several of these examples are very similar except for the input value and name. If you like you could create a default example template and the other examples can just pass in the additional props to the default template. See [here](https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/examples/search-field/SearchFieldExampleTemplate.jsx) as for an example.